package.json

```diff
@@ -7,23 +7,30 @@
     "test": "mocha"
   },
   "dependencies": {
-    "async": "^0.9.0",
-    "debug": "^2.1.1",
-    "loopback-connector": "^2.1.0",
+    "async": "^2.0.1",
+    "debug": "^2.2.0",
+    "loopback-connector": "^2.4.0",
     "mysql": "^2.5.4",
     "strong-globalize": "^2.5.8"
   },
   "devDependencies": {
-    "bluebird": "~2.9.10",
-    "loopback-datasource-juggler": "^2.28.0",
-    "mocha": "^2.1.0",
-    "rc": "^1.0.0",
-    "should": "^8.0.2",
-    "sinon": "^1.15.4"
+    "bluebird": "~3.4.1",
+    "eslint": "^2.13.1",
+    "eslint-config-loopback": "^4.0.0",
+    "loopback-datasource-juggler": "^2.47.0",
+    "mocha": "^3.0.1",
+    "rc": "^1.1.6",
+    "should": "^10.0.0",
+    "sinon": "^1.17.5"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/strongloop/loopback-connector-mysql.git"
   },
+  "scripts": {
+  "lint": "eslint .",
+  "test": "./node_modules/.bin/mocha -R spec --timeout 10000 test/*.js",
```

Comment:
You don't need `./node_modules/.bin`. Also remove `-R spec`, keep it simple unless we need it later on.

```
"test": "mocha --timeout 10000 test/*.js"